buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,7 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    jcenter()
+}
```

Comment:
Do we need this, couldn't we just remove it?

Reply:
I think that It's needed to obtain `org.jetbrains.kotlin:kotlin-compiler-embeddable `

Reply:
Oh, right, we need it for the DSL plugin itself. I'd still swap it for `mavenCentral()`, but that's a nitpick 😄